test/python/test_tools_import_special_phrases.py

```diff
@@ -11,6 +12,72 @@
 
 TEST_BASE_DIR = Path(__file__) / '..' / '..'
 
+def test_fetch_existing_words_phrases_basic(special_phrases_importer, word_table,
+                                            temp_db_conn):
+    """
+        Check for the fetch_existing_words_phrases() method.
+        It should return special phrase term added to the word
+        table.
+    """
+    with temp_db_conn.cursor() as temp_db_cursor:
+        query ="""
+            INSERT INTO word VALUES(99999, 'lookup_token', 'normalized_word',
+            'class', 'type', null, 0, 'near');
+        """
+        temp_db_cursor.execute(query)
+
+    assert not special_phrases_importer.words_phrases_to_delete
+    special_phrases_importer._fetch_existing_words_phrases()
+    contained_phrase = special_phrases_importer.words_phrases_to_delete.pop()
+    assert contained_phrase == ('normalized_word', 'class', 'type', 'near')
+
+def test_fetch_existing_words_phrases_housenumber(special_phrases_importer, word_table,
+                                                  temp_db_conn):
```

Comment:
You really should be using the temp_db_cursor fixture instead of creating the cursor yourself every time. The temp_db_conn fixture is only needed when you test a function that expects a connection as a parameter.